main.cpp

```diff
@@ -275,296 +300,281 @@ struct Bakery
         bool burn(int bakingTime);
     };
 
-    //bake bread
-    void bakeBread(double flourAmount, RyeBread brd);
-    //sell cakes
+    void bakeBread(double bFlourAmount, RyeBread brd);
     float cakeSell(std::string cakeName);
-    //smell great
     void smellGreat();
 
     RyeBread breadToSell;
 };
 
-/*
-Thing 3) aqua park
-5 properties:
-    1) number of pools (int)
-    2) number of saunas (int)
-    3) total length of water slides (double)
-    4) opening hours (float)
-    5) ticket price (float)
-3 things it can do:
-    1) relax muscles
-    2) be fun
-    3) learning swimming
- */
+void Bakery::bakeBread(double bFlourAmount, RyeBread brd)
+{
+    std::cout << "Flour used: " << bFlourAmount << std::endl << "Timer set to: " << brd.bakingTime << std::endl;
+}
+
+float Bakery::cakeSell(std::string cakeName)
+{
+    if (cakeName == "Sweet Cake")
+    {
+        std::cout << "Product sold: " << cakeName << std::endl;
+        return 9430.f;
+    }
+    
+    return 0.f;
+}
+
+void Bakery::smellGreat()
+{
+}
 
 struct AquaPark
 {
-    //number of pools (int)
     int numPools = 5;
-    //number of saunas (int)
     int numSaunas = 3;
-    //total length of water slides (double)
     float totalWaterSlidesLength = 121.56f;
-    //opening hours (float)
     float openAt = 9.30f;
-    //ticket price (float)
     float regularTicker = 13.21f;
 
-    //relax muscles
     void relaxMuscles(bool goToSauna, int swimmingIntensity);
-    //be fun
     void haveFun(float totalWaterSlidesLength, int numberOfSlides);
```

Comment:
Review our Slack discussion of using member variables in member functions directly. 
Change your argument list so that your function arguments are not duplicates of your member variables.
This can mean changing the function name (meaning the function purpose has changed) 
or it could mean changing the function arguments.
Remember, name things what they are or what they do.
- [ ] revise